python/handleInvalidState.py

```diff
@@ -83,7 +85,7 @@ def generateSafeHeadingDegrees(state, invalidStartObstacle):
     return globalWindDirectionDegrees
 
 
-def checkGoalValidity(state):
+def checkGoalValidity(state, goalLatlon=None):
```

Comment:
I like this solution! Could you add some documentation to the docstring right under here. Just say what the goalLatlon is and how it is used, and what happens with goalLatlon is None